fvm/src/gas/price_list.rs

```diff
@@ -945,9 +952,9 @@ impl PriceList {
 /// Returns gas price list by NetworkVersion for gas consumption.
 pub fn price_list_by_network_version(network_version: NetworkVersion) -> &'static PriceList {
     match network_version {
-        NetworkVersion::V18 | NetworkVersion::V19 | NetworkVersion::V20 => &HYGGE_PRICES,
+        NetworkVersion::V18 | NetworkVersion::V19 | NetworkVersion::V20 => &WATERMELON_PRICES,
```

Comment:
This is _wrong_, so I could drop support for nv18-20 in this PR if we'd like? Or we just do that separately, this is only going into the `next` staging branch anyway.

Reply:
Can you do this and the price-list rename in a prefactor?

Reply:
Done.